Gemfile.jruby-1.9.lock

```diff
@@ -126,12 +538,260 @@ GEM
     tins (1.3.4)
     treetop (1.4.15)
       polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot (>= 0.3.1)
```

Comment:
Don't know... and I don't have time to check this now, this is harmless so I will not try to fix it now but please go ahead if you want to try to fix it!